src/pkg/output/formatters.go

```diff
@@ -4,11 +4,23 @@ import (
 	"fmt"
 	"github.com/otterize/otterize-cli/src/pkg/cloudclient/restapi/cloudapi"
 	"github.com/samber/lo"
+	"golang.org/x/exp/constraints"
 	"strings"
 )
 
+func formatD[T constraints.Integer](c T) string {
```

Comment:
wut why? this just makes it harder to see that all it does is stringify the integer

Reply:
because there's so many uses that duplicate it and the code is horrible :(

Reply:
but it's fmt.Sprintf!
everyone knows what Sprintf does, and it's a single function call - which was replaced with another single function call, but then nobody knows what it does

Reply:
It was replaced with a shorter function call... I felt like it improves readability but I'm not feeling too strong about it, so reverting it if you insist.